tester/runner_test.go

```diff
@@ -16,6 +16,70 @@ import (
 	"github.com/open-policy-agent/opa/util/test"
 )
 
+func TestRunner_EnableFailureLine(t *testing.T) {
+
+	ctx := context.Background()
+
+	files := map[string]string{
+		"/a_test.rego": `package foo
+			test_a { 
+				true
+				false
+				true
+			}
+			test_b { 
+				false
+				true
+			}`,
+	}
+
+	tests := map[[2]string]struct {
+		wantErr  bool
+		wantFail bool
+		FailRow  int
+	}{
+		{"data.foo", "test_a"}: {false, true, 4},
+		{"data.foo", "test_b"}: {false, true, 8},
```

Comment:
Let's add a case to cover the single failure case that I described above.

Reply:
Done